src/containerapp/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 upcoming
 ++++++
+* Cloud Build Bugfix - 500 Internal Server Error (Wrong env selected to create builder)
```

Comment:
Please append related command `az containerapp up` in front of the description.

Reply:
Updated.